mm2src/coins_activation/src/platform_coin_with_tokens.rs

```diff
@@ -317,7 +378,7 @@ where
         mm_tokens.extend(tokens);
     }
 
-    let activation_result = platform_coin.get_activation_result(&req.request).await?;
+    let activation_result = platform_coin.get_activation_result(None, &req.request).await?;
```

Comment:
Is it ok to not use `task_handle` here?

Reply:
no, it is needed here (bug)

Reply:
well it's actually not a bug as we use cached accounts for re-enable (but still I have add task_handle for unification in [28bbba7](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2005/commits/28bbba74ca7320ead3fdbd946f35438303a2b952))